.travis.yml

```diff
@@ -20,11 +20,11 @@ before_install:
 
 install:
   - npm install -g bower
-  - npm install -g saucie
   - npm install
   - bower install
 
 before_script:
+  - curl -L https://gist.githubusercontent.com/johanneswuerbach/ce15304c6d33538ecc6e/raw/sauce-connect.sh | bash # Create a sauce tunnel
```

Comment:
Can we use the travis addon for opining the tunnel?
http://docs.travis-ci.com/user/sauce-connect/

```yml
addons:
  sauce_connect: true
```
That saves a download trip.